ReactCommon/jsi/sources.inc

```diff
@@ -14,22 +14,10 @@ SOURCES_V8_FILES = \
   ..\V8Runtime_droid.cpp \
   ..\V8Runtime_shared.cpp \
 
-SOURCES_JSC_FILES = \
-  ..\JSCRuntime.cpp \
```

Comment:
We have kept this because we wanted to keep the ability to switch b/w JS Engines.

Reply:
The jsc files were deleted from react native as part of RN 58.  It seems unlikely that we would readd the jsc impl over the V8 or other Js engines.

I am cleaning this up to unblock devmain which hit a break caused by the deletion of some of the jsc files.

Reply:
I am fine with cleaning jsc executor flow but JSCRuntime is present in latest RN and we should be able to conditionally fallback to it.

---
In reply to: [273331168](https://github.com/Microsoft/react-native/pull/44#discussion_r273331168) [](ancestors = 273331168)

Reply:
basically keep things as-is in ReactCommon\jsi

---
In reply to: [273334702](https://github.com/Microsoft/react-native/pull/44#discussion_r273334702) [](ancestors = 273334702,273331168)

Reply:
I am fine with cleaning jsc executor flow but JSCRuntime is present in latest RN and we should be able to conditionally fallback to it.
basically keep things as-is in ReactCommon\jsi